lib/tracker_api/resources/notification.rb

```diff
@@ -0,0 +1,15 @@
+module TrackerApi
+  module Resources
+    class Notification
+      include Virtus.model
+
+      attribute :id, Integer
+      attribute :message, String
+      attribute :kind, String
+      attribute :project_id, Integer
```

Comment:
You can change this to be a project and story object since the API returns nested objects.

```
attribute :project, , TrackerApi::Resources::Project
attribute :story, , TrackerApi::Resources::Stroy
attribute :performer, TrackerApi::Resources::Person
```

Then you can remove the code where you manually pull out `notification['story']['id']` and `notification['project']['id']`.